### DISCUSSION TITLE
Avalonia Opengl can not use GL_BLEND

### DISCUSSION THREAD

**Discussion by znoliver** [AvaloniaUI/Avalonia#17852] [Q&A] [answered]
### Describe the bug

I use Avalonia Opengl develop 3D application, and I want set some color has alpha, but I cant find setting like :glEnable(GL_BLEND);

### To Reproduce

None

### Avalonia version

11.2.2

**Comment by znoliver** [comment]
And glBlendFunc(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);

**Comment by maxkatz6** [comment, marked as answer]
GlInterface is implemented specifically for the Avalonia needs. If you need custom members, it shouldn't be difficult to use them. Silk.NET is a typical solution. Avalonia GLInterface provides GetProcAddress that allows to get a pointer to any GL method.
